src/snowflake/snowpark/context.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
+#
+
```

Comment:
I guess you also need a docstring for this new file (similar to https://github.com/snowflakedb/snowpark-python/blob/196cb4e9d3fd4a862bf07f55c9c16433925f3b49/src/snowflake/snowpark/udf.py#L5)